apis/fluentd/v1alpha1/plugins/tls_types.go

```diff
@@ -6,24 +6,14 @@ import (
 
 // Fluentd provides integrated support for Transport Layer Security (TLS) and it predecessor Secure Sockets Layer (SSL) respectively.
 type TLS struct {
-	// Force certificate validation
-	Verify *bool `json:"verify,omitempty"`
-	// Set TLS debug verbosity level.
-	// It accept the following values: 0 (No debug), 1 (Error), 2 (State change), 3 (Informational) and 4 Verbose
-	// +kubebuilder:validation:Enum:=0;1;2;3;4
-	Debug *int32 `json:"debug,omitempty"`
```

Comment:
This is a public document, is it necessary to remove these sections?

Reply:
It is not necessary, but it is not interpreted in the controller

Reply:
Ok, I understand your point of view, but if I undelete them, they will be automatically added to the CRD, you don't think it can be confusing ? what do you think about unlinking the loki struct from the *plugins.TLS struct ?

Reply:
https://docs.fluentd.org/configuration/transport-section
For the data structure of `tls` one should refer to the official documentation, so I don't think these should be removed, and as for any missing parts we will add them later.

Reply:
The tls docs for fluentd https://github.com/fluent/fluent-operator/blob/master/docs/plugins/fluentd/tls.md is copied from [fluentbit tls setting](https://docs.fluentbit.io/manual/pipeline/outputs/tcp-and-tls) which is incorrect.

As @wenchajun mentioned fluentd has its own [tls setting](https://docs.fluentd.org/configuration/transport-section) which is different with fluentbit's and should be optional to the fluentd input and output plugins: `The transport section must be under <match>, <source>, and <filter> sections. It specifies the transport protocol, version, and certificates.`

So fluentd's tls setting should conformed to fluentd's doc, not fluentbit's: https://docs.fluentd.org/configuration/transport-section 

Fluentd's tls setting should include params from 5 sections:
- https://docs.fluentd.org/configuration/transport-section#tls-setting
- https://docs.fluentd.org/configuration/transport-section#signed-public-ca-parameters
- https://docs.fluentd.org/configuration/transport-section#generated-and-signed-by-private-ca-parameters
- https://docs.fluentd.org/configuration/transport-section#generated-and-signed-by-private-ca-certs-or-self-signed-parameters
- https://docs.fluentd.org/configuration/transport-section#cert-digest-algorithm-parameter

Fluentd tls samples:
https://stackoverflow.com/questions/50190993/how-to-get-fluentd-td-agent-tls-ssl-encryption-for-in-forward-to-work
https://github.com/fluent/fluentd/issues/3802